ext/Logger.php

```diff
@@ -52,7 +54,7 @@ class Logger extends Extension
     /**
      * @var \Monolog\Logger
      */
-    protected $logger;
+    protected static $logger;
```

Comment:
I think there is no other way to access logger from other place rather than making it static. Probably it is ok as extensions are loaded in the very beginning of `run` so they are available almost everywhere. We have static `$output` inside `Codeception\Util\Debug` and I don't see any other way on implementing `codecept_debug` there